app/src/main/java/com/github/lookupgroup27/lookup/ui/overview/Landing.kt

```diff
@@ -1,5 +1,79 @@
-package com.github.lookupgroup27.lookup.ui.overview
-
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Home
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Icon
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import androidx.navigation.compose.rememberNavController
+import com.github.lookupgroup27.lookup.R
+
+/**
+ * LandingScreen displays the main landing page of the app. It includes a background image, logo,
+ * and a button with a home icon. The home button navigates to the "Menu" screen and the background
+ * is clickable to navigate to the "Map" screen.
+ *
+ * @param navController The NavController to handle navigation between screens.
+ */
+@Composable
+fun LandingScreen(navController: NavController) {
+  Box(
+      modifier = Modifier.fillMaxSize().clickable { navController.navigate("Map") },
```

Comment:
Better we use the constant TopLevelDestinations.MAP. from the navigation actions instead of the magic value "Map"

Reply:
I'll update the code to use TopLevelDestinations.MAP

Reply:
I also believe that you should use the navigateTo function (from NavigationActions).